src/compiler/bundle/bundle-output.ts

```diff
@@ -96,6 +96,10 @@ export const getRollupOptions = (
   const afterPlugins = config.rollupPlugins.after || [];
   const rollupOptions: RollupOptions = {
     input: bundleOpts.inputs,
+    output: {
+      // This option cannot be set if multiple inputs are provided
+      inlineDynamicImports: bundleOpts.inlineDynamicImports ? Object.keys(bundleOpts.inputs).length <= 1 : false,
```

Comment:
For my own edification, in what scenarios is `bundleOpts.inputs` equal to zero? This might be something worth adding a comment to